src/js/view/controls/components/chapters.mixin.js

```diff
@@ -35,15 +39,29 @@ const ChaptersMixin = {
         if (Array.isArray(data)) {
             data.forEach((obj) => this.addCue(obj));
             this.drawCues();
+            const existingCues = this._model.get('cues');
+            const playerCues = existingCues.concat(data);
+            this._model.set('cues', playerCues);
         }
     },
 
     chaptersFailed: function () {},
 
     addCue: function (obj) {
+        this.cues.forEach((cue) => {
+            if (cue.time === obj.begin) {
+                this.removeCue(cue);
+            }
+        });
```

Comment:
We didn't need to bother with duplicates before. Why do we now, and why is the chapter mixin responsible for this? Why are chapter cues more important than other cues?

Reply:
You told me last week that we don't want duplicate cues to appear if there are two cues set at the same time. Also, it's not necessarily chapters over ads - it's just what gets added first.

Reply:
I removed this. Will handle dupes in `drawCues()`.